srcpkgs/bismuth/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'bismuth'
+pkgname=bismuth
+version=3.1.4
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DUSE_TSC=OFF"
```

Comment:
i don't think setting build type is necessary. see 1af5a29d87a011232ae665d1c5f910f9b73588fd for more